contracts/axelar-gateway/src/event.rs

```diff
@@ -1,97 +1,34 @@
 use core::fmt::Debug;
 
-use cfg_if::cfg_if;
-use soroban_sdk::{Address, Bytes, BytesN, Env, IntoVal, String, Symbol, Topics, Val, Vec};
+use soroban_sdk::{Address, Bytes, BytesN, String};
+#[cfg(any(test, feature = "testutils"))]
```

Comment:
why is this needed for `Event` trait?

Reply:
But `emit` is being used in contract code which is part of the Event trait

Reply:
it was being used in EventTestutils matches impl. Just needed to import the trait there, fixed